app/helpers/application_helper.rb

```diff
@@ -77,4 +77,16 @@ def time_period_class(item)
     return 'currently-valid' if item.currently_valid?
     return 'expired' if item.expired?
   end
+
```

Comment:
Do you think announcement/achievements/assessments/etc index should use these helpers as well ?